piptools/resolver.py

```diff
@@ -448,3 +478,164 @@ def reverse_dependencies(
             ireq for ireq in ireqs if not (ireq.editable or is_url_requirement(ireq))
         ]
         return self.dependency_cache.reverse_dependencies(non_editable)
+
+
+class Resolver(BaseResolver):
+    """
+    Uses pip's 2020 resolver.
+    """
```

Comment:
Classes don't do things, they are not actions. They represent a state/concept/abstraction. You should describe what it is, not what it "does" because it doesn't do anything by itself, an external actor will call methods/actions on it.
```suggestion
    """A resolver wrapper that uses resolvelib."""
```

Reply:
Addressed in 81419f9f1090bda0761020e7e9832ba279a5fd4d